uiowa/uiowa
# Google CSE Results Styles

Resolves #1549 

## To Test
_Since search doesn't work locally, dev, stage_

- Pull and run a `blt frontend`
- Open up https://brand.prod.drupal.uiowa.edu/google-search?search=brand (normal google styles)
- Grab the compile css from here: docroot/themes/custom/uids_base/assets/css/components/global/search-results/search-results.css
- Inspect the brand URL from above and edit a style tag to paste in the copied css. Click out of the style tag to see the updated styles.
